sdk/digitaltwins/Azure.DigitalTwins.Core/tests/PublishTelemetryRecordedTestSanitizer.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Azure.Core.TestFramework;
+
+namespace Azure.DigitalTwins.Core.Tests
+{
+    public class PublishTelemetryRecordedTestSanitizer : RecordedTestSanitizer
```

Comment:
Good change! Can you add Pavel or just check with him once on how to handle GUIDs? Maybe this is something the test framework can support? I'm guessing other teams might need it too.

Reply:
Is the purpose of this change to ignore these headers in record matching? If you you can add them to ExcludedHeaders collection of record matcher (https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core.TestFramework/src/RecordMatcher.cs#L27).

We have support for generating Guids in the test framework:
https://github.com/azure/azure-sdk-for-net/blob/b9bd85cb2591f7c3f3c054f33f158e758056f1d3/sdk/core/Azure.Core.TestFramework/src/TestRandom.cs#L25

``` C#
Recording.Random.NewGuid()
```

Reply:
And by adding to collection I mean implement a matcher of you own that modifies the colleciton: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Infrastructure/FormRecognizerRecordMatcher.cs#L8 for example

Reply:
Thanks Sindhu and Pavel. I ended up using the Recording.Random.NewGuid utility method instead of sanitizing/excluding any headers.